src/Symfony/Component/Serializer/NameConverter/CamelCaseToSnakeCaseNameConverter.php

```diff
@@ -48,7 +48,7 @@ public function normalize($propertyName)
 
             $len = strlen($propertyName);
             for ($i = 0; $i < $len; ++$i) {
-                if (ctype_upper($propertyName[$i])) {
+                if (ctype_upper($propertyName[$i]) && $i !== 0) {
```

Comment:
What's the benefit to replace a call to `ctype_upper` by a call to `strtolower`?